### PR TITLE
Discussions topic updates DOC-1064, 1163

### PR DESCRIPTION
**Pull request by catong**
This PR addresses doc updates for DOC-1064 and DOC-1163.
Clarify the special chars supported in discussion topic IDs, add note warning against duplicate discussion topic names. (Also includes minor non-content edits and typo fixes)
@mhoeber, @explorerleslie @gwprice can you please review?

### CODE REVIEW

File: en_us/shared/building_and_running_chapters/running_course/discussions.rst [hunk 1]
```diff
@@ -94,13 +94,22 @@ Create Course-Wide Discussion Topics
...
-You can add more course-wide discussion topics to guide how students share and
-find information during your course. Such course-wide topics might include
-Feedback, Troubleshooting, or Technical Help. Discussions in these topics can
-begin as soon as your course is available.
+course-wide discussion topic named "General" is already included by default.
 
-To create a course-wide discussion topic:
+You can add additional course-wide discussion topics to guide how students
+share and find information during your course. Such course-wide topics might
+include Feedback, Troubleshooting, or Technical Help. Discussions in these
```
Comment (explorerleslie) [inline en_us/shared/building_and_running_chapters/running_course/discussions.rst]: I would pick either troubleshooting or technical help (they're basically the same thing) and add Introductions and Announcements.

**Comment by explorerleslie** [comment]
@catong One comment that should be addressed, otherwise :+1:

**Comment by gwprice** [comment]
:+1:

**Comment by mhoeber** [comment]
:+1:

**Comment by catong** [comment]
Merging https://github.com/edx/edx-documentation/pull/264

**Pull request merged**